code/index.html

```diff
@@ -3,28 +3,163 @@
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+            <div class="question-accordion" id="section4">
+              What will we eat during the trip?
+              </div>
+              <div class="answer">
+              <p>He lay on his armour-like back, and if he lifted his head a little he could see his brown belly, slightly domed and divided by arches into stiff sections.</p>
+              </div>
+
+              <div class="question-accordion" id="section5">
+                How far will we walk each day?
+                </div>
+                <div class="answer">
+                <p>He lay on his armour-like back, and if he lifted his head a little he could see his brown belly, slightly domed and divided by arches into stiff sections.</p>
+                </div>
+        </div>
+      </fieldset>
     </section>
+
+    <!-- Signup form -->
+    <div class="sign-up">
+      <fieldset>
+        <legend>Sign up!</legend>
+          <form method="POST" action="http://httpbin.org/anything">
+            <div class="questions">
+              <div class="name">
+                <label for="name">Name:</label>
+                <input 
+                type="text" 
+                id="name" 
+                name="name"
+                placeholder="Anna Svensson"
+                >
+              </div>
+  
+              <div class="question2">
+                <label for="question2">Email:</label>
+                <input 
+                type="text" 
+                id="question2" 
+                name="question2"
+                placeholder="xxxxxxx@xxx.se"
+                >
+              </div>
+          
+              <div class="question3">
+                <label for="question3">Phone number:</label>
+                <input 
+                type="text" 
+                id="question3" 
+                name="question3"
+                placeholder="xxx-xxxxxx"
+                >
+              </div>
+            </div>
+            
+
+            <div class="trips">
+              <label for="cars">I would like to come along to the trip:</label>
+              <select id="trips" name="trips">
+                <option value="Kebnekajse">Kebnekajse</option>
+                <option value="hagadalen">Hågadalen</option>
+                <option value="jamtlandstriangeln">Jämtlandstriangeln</option>
+                <option value="Padjelanta">Padjelanta</option>
+              </select>
+            </div>
+
+            
+              <div class="radio-button">
+                <p>How would you like to spend the nights?</p>
+                  <div class="radio" >
+                    <input type="radio" id="tent" name="tent" value="tent">
+                    <label for="Something1">In a tent</label><br>
+
+                    <input type="radio" id="cabin" name="Something" value="cabin">
+                    <label for="Something2">In a cabin</label><br>
```

Comment:
I think some of the labels doesn't match with the ID they are pointing at. I'm not sure that is why I can choose to both stay in a tent and a cabin (or a hotel). Maybe something to look into?
I get following message"The for attribute of the label element must refer to a non-hidden form control."